### ISSUE TITLE
Refactor all the events

### ISSUE THREAD

**Issue by sinkingsugar** [fragcolor-xyz/clamor#85]
Currently we use anonymous tuples, move to something clearer like:
```rust
MetadataSet {
	asset_id: T::AssetId,
	name: Vec<u8>,
	symbol: Vec<u8>,
	decimals: u8,
	is_frozen: bool,
	can_transfer: bool,
},
```

**Comment by sinkingsugar** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Whole-Team-623c70cfcdeee40011bd03e3/issues/fragcolor-xyz/clamor/85?planning-poker) with ZenHub @alebaffa @aang114

**Comment by alebaffa** [comment]
@sinkingsugar Do you mean to refactor events like this

`Self::deposit_event(Event::InventoryAdded(to, class, (edition, copy)));`

to something like
`Self::deposit_event(Event::InventoryAdded(MetadataSet));`
?

**Comment by sinkingsugar** [comment]
No, I don't want new types. Just named anonymous type within the Enum itself.

**Comment by sinkingsugar** [comment]
Example @alebaffa https://github.com/fragcolor-xyz/protos/blob/2ae8ba0a96c6248f8f92514e43ca24eb0d6f38c1/src/traits.rs#L17-L23

**Comment by alebaffa** [comment]
Fixed in PR #100 .
